src/jobs/get-br-dependencies.yml

```diff
@@ -0,0 +1,76 @@
+parameters:
+  exec:
+    type: executor
+  aws-access-key-id:
+    type: env_var_name
+    default: AWS_ACCESS_KEY_ID
+  aws-secret-access-key:
+    type: env_var_name
+    default: AWS_SECRET_ACCESS_KEY
+  aws-region:
+    type: env_var_name
+    default: AWS_REGION
+  cache-bucket-uri:
+    type: env_var_name
+    default: CACHE_BUCKET_URI
+  resource-class:
+    type: string
+    default: medium
+  hex-validate:
+    type: boolean
+    default: true
+  env-setup:
+    type: steps
+    default: []
+  save-dl-cache:
+    type: boolean
+    default: true
+  save-public-cache:
+    type: boolean
+    default: false
+executor: << parameters.exec >>
+resource_class: << parameters.resource-class >>
+steps:
+  - checkout
+  - steps: << parameters.env-setup >>
+  - install-elixir:
+      version: $ELIXIR_VERSION
+  - install-hex-rebar
+  - install-nerves-bootstrap
+  - run:
+      name: Install AWS CLI
+      command: |
+        apt install -y python3-pip
+        pip install awscli
+
+        aws configure set aws_access_key_id ${<< parameters.aws-access-key-id >>}
+        aws configure set aws_secret_access_key ${<< parameters.aws-secret-access-key >>}
+        aws configure set region ${<< parameters.aws-region >>}
```

Comment:
Should this be conditional setup step as well since the ability to save to public S3 is conditional below? What will happen if the user does not provide these settings?

Reply:
Good call. Updated in 3edb692.